sql/org_mozilla_firefox_derived/incline_executive_v1/query.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TEMP FUNCTION bucket_manufacturer(manufacturer STRING) AS (
+  IF(
+    manufacturer IN UNNEST(['samsung', 'huawei', 'xiaomi', 'lge', 'motorola', 'sony', 'google', 'oppo', 'oneplus']),
+    manufacturer,
+    'Other')
+);
+
+CREATE TEMP FUNCTION bucket_country(country STRING) AS (
+  IF(
+    country IN UNNEST(['US', 'CA', 'DE', 'IN', 'FR', 'CN', 'IR', 'BR', 'IE', 'GB', 'ID']),
+    [country, 'Tier 1'],
+    ['Other'])
+);
+
+WITH
+  fennec_client_info AS (
+  SELECT
+    clients_last_seen.submission_date AS date,
+    IF(migrated_clients.fenix_client_id IS NOT NULL, 'Yes', 'No') AS is_migrated,
+    app_name,
+    clients_last_seen.normalized_channel AS channel,
+    SPLIT(device, '-')[OFFSET(0)] AS manufacturer,
+    country,
+    `moz-fx-data-shared-prod.udf.active_n_weeks_ago`(days_seen_bits,
+      0) AS active_this_week,
+    `moz-fx-data-shared-prod.udf.active_n_weeks_ago`(days_seen_bits,
+      1) AS active_last_week,
+    `moz-fx-data-shared-prod.udf.active_n_weeks_ago`(days_created_profile_bits,
+      0) AS new_this_week,
+    `moz-fx-data-shared-prod.udf.active_n_weeks_ago`(days_created_profile_bits,
+      1) AS new_last_week
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.core_clients_last_seen_v1` clients_last_seen
+  LEFT JOIN
+    `moz-fx-data-shared-prod.org_mozilla_firefox.migrated_clients_v1` migrated_clients
+    ON clients_last_seen.client_id = migrated_clients.fennec_client_id
+  WHERE
+    AND clients_last_seen.submission_date = @submission_date
+    AND migrated_clients.submission_date >= @submission_date
+    AND app_name = 'Fennec'
+    AND os = 'Android' ),
+  fenix_client_info AS (
+  SELECT
+    clients_last_seen.submission_date AS date,
+    IF(migrated_clients.fenix_client_id IS NOT NULL, 'Yes', 'No') AS is_migrated,
+    'Fenix' as app_name,
+    clients_last_seen.normalized_channel AS channel,
+    baseline.device_manufacturer AS manufacturer,
+    baseline.country,
+    `moz-fx-data-shared-prod.udf.active_n_weeks_ago`(baseline.days_seen_bits,
+      0) AS active_this_week,
+    `moz-fx-data-shared-prod.udf.active_n_weeks_ago`(baseline.days_seen_bits,
+      1) AS active_last_week,
+    DATE_DIFF(clients_last_seen.submission_date, baseline.first_run_date, DAY) BETWEEN 0 AND 6  AS new_this_week,
+    DATE_DIFF(clients_last_seen.submission_date, baseline.first_run_date, DAY) BETWEEN 7 AND 13 AS new_last_week
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_firefox_derived.clients_last_seen_v1` clients_last_seen
```

Comment:
I'd recommend using the `org_mozilla_firefox.clients_last_seen` view instead, which will avoid you having to descend into `baseline`.